state/git/impl.go

```diff
@@ -47,6 +52,20 @@ type JSONGitStore struct {
 
 func NewJSONGitStore(o *JSONGitStoreOptions) *JSONGitStore {
```

Comment:
@kris-nova My question lies in here how to pass the git object configurations in the program. I need some sample data